tools/test_default_firewall_ns.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/python3
+
+import sys
+import json
+from cloudcix_primitives import default_firewall_ns
+
+# Prerequisites for running this test script:
+#
+#   tools/test_ns.py build ns1100
+#   tools/test_bridgeif_ns.py build br-B1 ns1100
+
+# Fetch command and arguments
+cmd = sys.argv[1] if len(sys.argv) > 1 else None
+public_bridge = "br-B1"
+namespace_name = "ns1100"
+config_file = "/etc/cloudcix/pod/configs/config.json"
+
+if len(sys.argv) > 2:
+    namespace_name = sys.argv[2]
+if len(sys.argv) > 3:
+    bridgename = sys.argv[3]
```

Comment:
Should this be as such?
```suggestion
    public_bridge = sys.argv[3]
```

Reply:
One could do that, but no matter how trivial a cosmetic change this may seem, I'd have to set aside the `nat_firewall_ns primitive` I'm currently working on, make the change, test everything again to make sure I didn't make a typo that breaks this - already tested and known to work - code and subsequently rebase all branches that depend on this primitive (which is all branches for primitives). I'd prefer not to do that if it can be helped.

Maybe we can address cosmetic changes such as this one post-merge, i.e. I'd appreciate if you could create an issue to be tackled at a later point. On a more general note, it's probably a good idea to create a cleanup issue to collect such things in anyway, for there are plenty of other, even rougher spots already in the code base.

Reply:
This isnt meant to be a cosmetic change, I just dont see `bridgename` being referenced in this file, so I assumed you meant to set the `public_bridge` variable with its default value above on L14

Reply:
Done. Also, I fixed a rather more serious problem with the rule set (only spotted that one when I diffed the rule set resulting from `nat_firewall_ns.build()` against a blank rule set created by `default_firewall_ns.build()`.